packages/core/test/integration/query-interface.test.js

```diff
@@ -585,11 +586,15 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
           });
           let constraints = await this.queryInterface.showConstraint('users');
           constraints = constraints.map(constraint => constraint.constraintName);
-          expect(constraints).to.include('check_user_roles');
-          await this.queryInterface.removeConstraint('users', 'check_user_roles');
-          constraints = await this.queryInterface.showConstraint('users');
-          constraints = constraints.map(constraint => constraint.constraintName);
-          expect(constraints).to.not.include('check_user_roles');
+          if (dialectName === 'mysql' && semver.lt(current.getDatabaseVersion(), '8.0.16')) {
+            // MySQL 8.0.16+ has a new INFORMATION_SCHEMA.CHECK_CONSTRAINTS table
+            expect(constraints).to.not.include('check_user_roles');
```

Comment:
I've added this back in

Reply:
This will part of not be part of showConstraint if the dialect is MySQL and the version is < 8.0.16 as check constraints are not supported in older versions.